package/rpm/SOURCES/etc/init.d/sth

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+# Load some fancy functions for init.d
+. /etc/rc.d/init.d/functions
+
+PARAM=$1
+
+STH_INSTANCE=${2}
+
+COMPONENT_NAME=sth
+PREFIX=/usr
+COMPONENT_DIR=${PREFIX}/${COMPONENT_NAME}
+CONFIG_FILE=${PREFIX}/${COMPONENT_NAME}/conf/sth.conf
+COMPONENT_USER=sth
+
+NODE_EXEC="/usr/bin/node"
+
+component_start()
```

Comment:
I think that checking MongoDB connection is not the purpose of the init.d script.Init script should check only if any instance of the component is running to _start_, _stop_ or _status_ it. If no MongoDB connection is possible STH should crash and write it into logfile. Init script only write OK or FAIL if STH starts successfully or not.